freshmaker/lightblue.py

```diff
@@ -581,9 +581,9 @@ def _make_request(self, entity, data):
                 record_mode=conf.vcrpy_mode,
             )
             with my_vcr.use_cassette(f'{self.event_id}.yml'):
-                response = requests.post(entity_url, **request_kwargs)
+                response = requests.post(entity_url, **request_kwargs, timeout=conf.requests_timeout)
         else:
-            response = requests.post(entity_url, **request_kwargs)
+            response = requests.post(entity_url, **request_kwargs, timeout=conf.requests_timeout)
```

Comment:
Requests to LightBlue can take much more time, so we don't want to apply the same timeout to LightBlue requests as others, I believe some LightBlue requests can even take more than 5mins to finish.

Reply:
@qixiang I fixed the timeout for LightBlue. Review one more time, please.